### PR TITLE
87 update actions for GitHub flow

### PR DESCRIPTION
**Pull request by cjbatin**
## Ticket name and link

#87 

## Preview

No Visual Changes

## Description

Makes deployments to development on merge into main. Deployments to production can be run manually and require approval

### CODE REVIEW

File: .github/workflows/development.yml [hunk 1]
```diff
@@ -1,9 +1,10 @@
-name: develop
+name: development
 
 on:
   push:
     branches:
-      - develop
+      - main
+      - 87-Update-Actions-For-Github-Flow
```
Comment (Jake-Sturgeon) [inline .github/workflows/development.yml:7]: Will this be removed later?

File: .github/workflows/distribute-apps.yml [hunk 2]
```diff
@@ -7,36 +7,41 @@ on:
...
+        uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
           node-version-file: '.node-version'
           cache: yarn
       - uses: ruby/setup-ruby@v1
       - uses: maxim-lobanov/setup-xcode@v1
         with:
-          xcode-version: '15.2'
+          xcode-version: '16.0'
```
Comment (Jake-Sturgeon) [inline .github/workflows/distribute-apps.yml:35]: Will this require everyone to be running 16+

File: .github/workflows/development.yml (hunk 1 above)
Comment (cjbatin) [reply .github/workflows/development.yml:7]: Yeah, I'm going to remove it now

File: .github/workflows/distribute-apps.yml (hunk 2 above)
Comment (cjbatin) [reply .github/workflows/distribute-apps.yml:35]: I don't believe it should

**Review by Jake-Sturgeon — APPROVED** [review]

**Pull request merged**